tests/e2e/dsc_creation_test.go

```diff
@@ -12,7 +13,7 @@ import (
 	operatorv1 "github.com/openshift/api/operator/v1"
 	"github.com/stretchr/testify/require"
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
should it be `apierrs` like in other places?

Reply:
Seen both. Will check other places and align in a follow up PR.

Reply:
@ykaliuta I opened a follow-up PR #1078 1078